### PR TITLE
updated readme of the imitation learning to include references and authors names

### PR DESCRIPTION
**Pull request by mostafaelaraby**

### CODE REVIEW

**Comment by bhairavmehta95** [comment]
Please fix grammar issues (you can use grammarly.com) and use the actual BibTeX for @manfreddiaz 's thesis

**Comment by mostafaelaraby** [comment]
Updated readme with bibtex and fixed grammar issues

File: learning/imitation/iil-dagger/README.md [hunk 1]
```diff
@@ -1,28 +1,43 @@
...
+$ python -m learning.imitation.iil-dagger.train
 
 5) Test the trained agent specifying the saved model:
 
-    $ python -m learning.imitation.pytorch-v2.test --model-path ![path]
-
+$ python -m learning.imitation.pytorch-v2.test --model-path ![path]
+
+## Acknowledgment
+- We started from previous work done by Manfred Díaz as a boilerplate, and we would like to thank him for his full support with code and answering our questions
```
Comment (bhairavmehta95) [inline learning/imitation/iil-dagger/README.md]: Add period at the end

File: learning/imitation/iil-dagger/README.md [hunk 2]
```diff
@@ -1,28 +1,43 @@
...
+
+## Authors
+- [Mostafa ElAraby ](https://www.mostafaelaraby.com/)
+- [Linkedin](https://linkedin.com/in/mostafaelaraby)
+- Ramon Emiliani
+- [Linkedin](https://www.linkedin.com/in/ramonemiliani)
+## References
+@phdthesis{diaz2018interactive,
```
Comment (bhairavmehta95) [inline learning/imitation/iil-dagger/README.md]: add new line, and make it formatted:

## References

To cite this project, please use:

```
@phdthesis{diaz2018interactive,
  title={Interactive and Uncertainty-aware Imitation Learning: Theory and Applications},
  author={Diaz Cabrera, Manfred Ramon},
  year={2018},
  school={Concordia University}
}
```

File: learning/imitation/iil-dagger/README.md [hunk 3]
```diff
@@ -1,28 +1,43 @@
...
+$ python -m learning.imitation.pytorch-v2.test --model-path ![path]
+
+## Acknowledgment
+- We started from previous work done by Manfred Díaz as a boilerplate, and we would like to thank him for his full support with code and answering our questions
+
+## Authors
+- [Mostafa ElAraby ](https://www.mostafaelaraby.com/)
+- [Linkedin](https://linkedin.com/in/mostafaelaraby)
+- Ramon Emiliani
+- [Linkedin](https://www.linkedin.com/in/ramonemiliani)
```
Comment (bhairavmehta95) [inline learning/imitation/iil-dagger/README.md]: new line between Ramon + ##References

File: learning/imitation/iil-dagger/README.md [hunk 4]
```diff
@@ -1,28 +1,43 @@
-# Imitation Learning using Dataset Aggregation
+# Imitation Learning
 
 ## Introduction
-In this baseline we train a small squeezenet model on expert trajectories to simply clone the behaviour of the expert.
-Using only the expert trajectories would result in a model unable to recover from non-optimal positions ,Hence we use a technique called DAgger a dataset aggregation technique with mixed policies between expert and model.
+In this baseline we train a small squeezenet model on expert trajectories to simply clone the behavior of the expert.
```
Comment (bhairavmehta95) [inline learning/imitation/iil-dagger/README.md]: Cite Squeezenet and DAgger, or atleast refer to them

File: learning/imitation/iil-dagger/README.md [hunk 5]
```diff
@@ -1,28 +1,43 @@
...
+Using only the expert trajectories would result in a model unable to recover from non-optimal positions,Hence we use a technique called DAgger a dataset aggregation technique with mixed policies between expert and model.
```
Comment (bhairavmehta95) [inline learning/imitation/iil-dagger/README.md]: Using only the expert trajectories would result in a model unable to recover from non-optimal positions; Instead, we use a technique called DAgger: a dataset aggregation technique with mixed policies between expert and model.

**Pull request merged**